src/main.c

```diff
@@ -1,9 +1,8 @@
-#include <stdlib.h>
+#include "board_definitions.h"
+#include "hardware_functions.h"
 #include "SPI.h"
```

Comment:
iets met #ifdef LORA_BOARD voor deze libraries?

Reply:
is nu gefikst in lora.c met een warning